docs/docs/configuration/hardware_acceleration.md

```diff
@@ -161,3 +161,51 @@ processes:
 If you do not see these processes, check the `docker logs` for the container and look for decoding errors.
 
 These instructions were originally based on the [Jellyfin documentation](https://jellyfin.org/docs/general/administration/hardware-acceleration.html#nvidia-hardware-acceleration-on-docker-linux).
+
+### NVIDIA Jetson (Nano, NX, AGX)
```

Comment:
Just curious, does jetson support nvidia-smi?

Reply:
No, it doesn’t. It does have its own version of the metrics, so we can still get GPU load and memory. I’ll probably add that after enabling tensorrt. Unfortunately the accelerator (encoder/decoder, DLA, VIC, PVA) metrics don’t include load.